src/main/java/uk/gov/hmcts/ccd/security/idam/IdamRepository.java

```diff
@@ -35,7 +39,7 @@ public UserInfo getUserInfo(String jwtToken) {
 
     @Cacheable("idamUserRoleCache")
     public List<String> getUserRoles(String userId) {
-        String dataStoreSystemUserToken = getDataStoreSystemUserAccessToken();
+        String dataStoreSystemUserToken = selfInstance.getDataStoreSystemUserAccessToken();
         List<String> roles = getUserByUserId(userId, dataStoreSystemUserToken).getRoles();
         log.debug("System user queried user info from IDAM API. User Id={}. Roles={}.", userId, roles);
```

Comment:
@paulridout , @VamshiRudrabhatlaHMCTS  - Please let's make sure that this log line is printing this line in preview environment pods of the PR. That will ensure the mechanism worked well. Then we can release this.

Reply:
Altered logging level to debug for uk.gov.hmcts.ccd.security.idam package.  After this saw examples of the debug message in the logs, e.g.
`2022-02-17T13:41:30.651 DEBUG [http-nio-4452-exec-2] uk.gov.hmcts.ccd.security.idam.IdamRepository System user queried user info from IDAM API. User Id=<excluded>. Roles=[caseworker-befta_master, caseworker-publiclaw, caseworker-divorce, caseworker-divorce-solicitor, caseworker-probate, caseworker-probate-solicitor, caseworker-ia, caseworker-publiclaw-solicitor, pui-case-manager, caseworker-divorce-financialremedy, caseworker-befta_master-solicitor, caseworker-ia-legalrep-solicitor, caseworker-divorce-financialremedy-solicitor, caseworker].`